homeassistant/helpers/condition.py

```diff
@@ -34,7 +34,14 @@ def from_config(config: ConfigType, config_validation: bool=True):
         raise HomeAssistantError('Invalid condition "{}" specified {}'.format(
             config.get(CONF_CONDITION), config))
 
-    return factory(config, config_validation)
+    invert = config.get(CONF_INVERT) or False
```

Comment:
And yes, each method will need it's own test. I remember suggesting you start with the `state` condition for exactly that reason 👍